spigot/pom.xml

```diff
@@ -91,22 +91,22 @@
             <scope>compile</scope>
         </dependency>
         <dependency>
-            <groupId>com.destroystokyo.paper</groupId>
```

Comment:
Can you please revert this change? This is not needed and will break pre 1.20 setups.

Reply:
Done. Thank you!

Reply:
> Can you please revert this change? This is not needed and will break pre 1.20 setups.

Done. Thank you!